OpenEdition Journals.js

```diff
@@ -148,32 +163,36 @@ function doWeb(doc, url) {
 			}
 		} */
 
-		Zotero.selectItems(ZU.getItemArray(doc, results), function(selectedItems) {
+		Zotero.selectItems(ZU.getItemArray(doc, results), function (selectedItems) {
 			if (!selectedItems) return true;
 
-			var urls = new Array();
-			for (var i in selectedItems) {
+			var urls = [];
+			for (let i in selectedItems) {
 				urls.push(i);
 			}
 
-			ZU.processDocuments(urls, function(doc) {
-				scrape(doc, doc.location.href)
+			ZU.processDocuments(urls, function (doc) {
```

Comment:
this should just work as `ZU.processDocuments(urls, scrape)` does it not?

Reply:
Yes, it does